composer.json

```diff
@@ -23,9 +23,9 @@
         "ext-curl": "*",
         "ext-json": "*",
         "ext-zip": "*",
-        "neitanod/forceutf8": "^1.4.0 || ^2.0.0",
-        "symfony/finder": "^2.5.0 || ^3.0.0 || ^4.0.0",
-        "symfony/http-foundation": "^2.5.0 || ^3.0.0 || ^4.0.0"
+        "neitanod/forceutf8": "^1.4 || ^2.0",
```

Comment:
```suggestion
        "neitanod/forceutf8": "^2.0",
```

Reply:
Please let's do this in another PR. This PR is only about making the requirements shorter, not about raising them.